py3status/modules/scratchpad_counter.py

```diff
@@ -4,10 +4,10 @@
 
 Configuration parameters:
     cache_timeout: How often we refresh this module in seconds (default 5)
-    format: Format of indicator. {} replaces with count of windows
-        (default '{} ⌫')
-    hide_when_none: Hide indicator when there is no windows (default False)
```

Comment:
> Remove docstring, but keep the backward compatible code?

Not sure that is actually possible because the tests would complain.

Just keep it how it was.